data/build.gradle

```diff
@@ -20,8 +20,8 @@ android {
 	}
 
 	compileOptions {
-		sourceCompatibility JavaVersion.VERSION_1_8
-		targetCompatibility JavaVersion.VERSION_1_8
+		sourceCompatibility JavaVersion.VERSION_17
+		targetCompatibility JavaVersion.VERSION_17
```

Comment:
Since API level 34 is the first to support Java 17 and since this is quite a jump: Are all required APIs available through desugaring down to API level 26?

Reply:
Our new Gradle Plugin version requires Java 17 https://developer.android.com/build/releases/gradle-plugin#compatibility. The Gradle Plugin version is required for Kotlin etc etc...There is no other possibility than targeting Java 17.